src/server/game/Server/WorldSession.h

```diff
@@ -550,6 +560,9 @@ class WorldSession
         void SendAuctionOwnerNotification(AuctionEntry* auction);
         void SendAuctionRemovedNotification(uint32 auctionId, uint32 itemEntry, int32 randomPropertyId);
 
+        // Black Market
+        void SendBlackMarketHello(ObjectGuid Guid, Creature* unit);
```

Comment:
change `Guid` to `guid`
the 2nd argument is a unit or a creature? if this is a pointer to the auctioneer rename `unit` to `auctioneer`